src/effects/RealtimeEffectState.cpp

```diff
@@ -213,20 +213,25 @@ struct RealtimeEffectState::Access final : EffectSettingsAccess {
    std::weak_ptr<RealtimeEffectState> mwState;
 };
 
-RealtimeEffectState::RealtimeEffectState(const PluginID & id)
+RealtimeEffectState::RealtimeEffectState(const PluginID& id)
 {
    SetID(id);
 }
 
 RealtimeEffectState::RealtimeEffectState(const RealtimeEffectState &other)
-  : mID{ other.mID }
+  : mEffectStateExtensions(other.mEffectStateExtensions)
+  , mID{ other.mID }
   , mPlugin{ other.mPlugin }
   , mMainSettings{ other.mMainSettings }
+  , mwInstance { other.mwInstance }
```

Comment:
This member should move earlier in the header file, before `mMainSettings` (and the initializers ordered appropriately) just so that the `@name` section comments are correct.